src/index.html

```diff
@@ -102,5 +102,6 @@
     <div class="skinHeader focuscontainer-x"></div>
     <div class="mainAnimatedPages skinBody"></div>
     <div class="mainDrawerHandle"></div>
+    <%= htmlWebpackPlugin.options.appLoader %>
```

Comment:
Won't this break the (future) ability to serve those files as static ones by e.g. nginx? Can we change the logic so that it's possible to do so, like this?

```suggestion
    <!-- <%= htmlWebpackPlugin.options.appLoader %> -->
```

then you won't need to change the server...

Reply:
Yeah, I am 100% in favor of serving the web client with webpack, but not in a way that requires more server interaction.

Reply:
I understand, and i've also tried to use the regular mechanism of the html-webpack-plugin. 
However, i don't have enough insight on how the current build-process is implemented.

In a "standard" webpack setup (at least from projects i have been working on), the entry-point is compiled into a single js filed and injected into the final html file.

This does not seem to be the case here. The "dist" folder has basically a copy of all the assets in the "src" folder after a "npm build" (with the webpack entry-point bundle.js being the only minified file").

Basically, the only thing to make webpack-dev serving work is to add the apploader.js script into the index.html. I think the only built-in webpack way is to do it with the html-webpack-plugin, but not with the standard mechanism which injects the output bundle.js, because we need a different script in the index.html.

The other option is to create a webpack plugin which simply injects the script tag hard-coded.

Do you have any other suggestions? I'd be happy to work on them, but i think without a proper local developer setup, contributors will have a hard time working on this project.

Reply:
> The other option is to create a webpack plugin which simply injects the script tag hard-coded.

This seems like a decent solution until we can build the entire source tree with webpack, which is still a work in progress.

Reply:
@dkanada found a way to inject the apploader unmanipulated using the webpack-concat-plugin plugin.